app/collins/models/conversions.scala

```diff
@@ -136,7 +137,10 @@ object conversions {
       (json \ "UPDATED").asOpt[Timestamp],
       (json \ "DELETED").asOpt[Timestamp],
       (json \ "ID").as[Long],
-      (json \ "STATE").asOpt[State].map(_.id).getOrElse(0)))
+      (json \ "STATE") match {
+        case JsNull => State.ANY_STATUS
```

Comment:
I think you might be confusing Status (`New`, `Incomplete`, `Allocated` etc.) and State (`RUNNING`, `MAINT_NOOP` etc). A newly created asset will have a status, but not a state.